mmv1/third_party/terraform/services/resourcemanager/data_source_google_service_accounts.go

```diff
@@ -17,10 +18,18 @@ func DataSourceGoogleServiceAccounts() *schema.Resource {
 	return &schema.Resource{
 		Read: datasourceGoogleServiceAccountsRead,
 		Schema: map[string]*schema.Schema{
+			"prefix": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
 			"project": {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"regex": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
nit, you could add a validation func here to ensure the REGEX compiles. 

https://github.com/hashicorp/terraform-plugin-sdk/blob/b11308dc8f0ff4a03f750ba012e7bc7d3974e895/helper/schema/schema.go#L342-L349

Reply:
It's a good suggestion.
For me, to check, if a regex compiles is a repeatable task.

I'd like to add a function to the [`github.com/hashicorp/terraform-provider-google/google/verify`](https://github.com/hashicorp/terraform-provider-google/blob/main/google/verify/validation.go) package and use it here.